src/jdk.incubator.foreign/share/classes/jdk/incubator/foreign/CLinker.java

```diff
@@ -72,7 +415,7 @@ public static ForeignLinker getSystemLinker() {
      * As such, this interface only supports reading {@code int}, {@code double},
      * and any other type that fits into a {@code long}.
      */
-    public interface VaList extends Addressable, AutoCloseable {
+    interface VaList extends Addressable, AutoCloseable {
```

Comment:
Comment about sealedness and immutability here - probably for the Builder too

Reply:
Can do sealed, but neither VaList and Builder impls are immutable.

Reply:
ah sure - just sealed